ubuntu/Dockerfile

```diff
@@ -13,9 +13,13 @@ RUN cd /tmp \
  && cmake /usr/src/root \
 	-Dall=ON \
 	-Dcxx14=ON \
-	-Dfail-on-missing=ON \
-	-Dgnuinstall=ON \
+	-Dfail-on-missing=OFF \
```

Comment:
We need to keep `-Dfail-on-missing=ON` here. It's used to catch missing dependencies in the Dockerfile.

Reply:
fixed